lib/bookmarkable.rb

```diff
@@ -6,6 +6,7 @@ def self.included(bookmarkable)
       has_many :user_tags, through: :bookmarks, source: :tags
       after_update :update_bookmarks_index
       after_update :update_bookmarker_pseuds_index
+      after_destroy :update_bookmarker_pseuds_index
```

Comment:
Just thinking out loud here for the sake of recording it for posterity, but at some point, we might need/want to do `after_destroy :update_bookmarks_index` here. A recent conversation revealed that if I bookmark a work tagged "The X-Files" and the work is deleted, the bookmark will continue to appear on the bookmark page for "The X-Files"... _but_ if "The X-Files" later gets synned to "The X-Files (TV 1993)", my bookmark will (understandably) disappear.

Reply:
In theory, the separation of the bookmarks index into bookmark documents and bookmarkable documents should make that issue disappear once the new search code is deployed.

Because the work's filters are no longer being stored in the bookmark's document, the relationship between the bookmark and the tag "The X-Files" ought to disappear as soon as the work disappears from the bookmarkable index. (And because the Work class includes `lib/searchable.rb`, it has an `after_destroy` hook that queues it up for reindexing.)